src/popup/router/components/Input.vue

```diff
@@ -20,11 +20,7 @@ export default {
   created() {},
   watch: {
     value(val) {
-      if ((this.type === 'number' && isNaN(val)) || parseFloat(val) === 0) {
```

Comment:
I don't see reasons to check is `parseFloat(val) === 0` if input type is not `number`.